internal/executor/thread-mate/config.go

```diff
@@ -56,6 +56,11 @@ func (c *Config) Validate() error {
 // MergeConfigs merges the configuration.
 func MergeConfigs(configs []*executor.Config) (Config, error) {
 	defaults := Config{
+		Logger: config.Logger{
+			Level:         "info",
```

Comment:
Doesn't look configurable, but I'll leave it up to you if it needs to be configurable or not.

Reply:
It's just a default, other values can be specified in the YAML config :)